src/main/modules/Execution.test.js

```diff
@@ -0,0 +1,8 @@
+import { Execution } from './Execution.js';
+
+var parserActionssArray = [{'actionActor': 'Marco', 'actionType': 'Pick Weapon', 'element': 'Handgun'}, {'actionActor': 'Marco', 'actionType': 'Heal', 'element': 5}, {'actionActor': 'RAT1', 'actionType': 'Receive Attack', 'fromActor': 'Marco'}, {'actionActor': 'Marco', 'actionType': 'Shoot Weapon', 'element': 'Shotgun'}, {'actionActor': 'Marco', 'actionType': 'Receive Attack', 'element': 'RAS1'}, {'actionActor': 'RAS1', 'actionType': 'Receive Attack', 'element': 'Marco'}, {}];
```

Comment:
In order to make the code more readable, I suggest you to create this kind of variables on this way:

```
var parserActionssArray = [
  {'actionActor': 'Marco', 'actionType': 'Pick Weapon', 'element': 'Handgun'}, 
  {'actionActor': 'Marco', 'actionType': 'Heal', 'element': 5}, 
  {'actionActor': 'RAT1', 'actionType': 'Receive Attack', 'fromActor': 'Marco'}, 
  {'actionActor': 'Marco', 'actionType': 'Shoot Weapon', 'element': 'Shotgun'}, 
  {'actionActor': 'Marco', 'actionType': 'Receive Attack', 'element': 'RAS1'}, 
  {'actionActor': 'RAS1', 'actionType': 'Receive Attack', 'element': 'Marco'}, 
  {}];
```

Reply:
Good apreciation, these changes were already implemented, check them and tell me what you think.